src/Grids/vertically_stretched_rectilinear_grid.jl

```diff
@@ -108,6 +99,12 @@ function VerticallyStretchedRectilinearGrid(FT=Float64, architecture=CPU();
     # operator wanting to access ΔzC[Nz+2].
     ΔzC = OffsetArray(cat(ΔzC[0], ΔzC..., ΔzC[Nz], dims=1), -Hz-1)
 
+    # Convert to appropriate array type for arch
+     zF = OffsetArray(arch_array(architecture,  zF.parent),  zF.offsets...)
```

Comment:
Search and replace `zF` to `zᵃᵃᶜ` etc? Or too much to ask?

Reply:
Will do!

Reply:
Done.